lib/tasks/heart_rate_data.rake

```diff
@@ -0,0 +1,8 @@
+namespace :heart_rate_data do
+  desc "Update heart rate data for each user for Fitbit"
+  task sync: :environment do
+    HeartRateData.all.each do |heart_rate_data|
+      heart_rate_data.update_heart_rate_data!
+    end
```

Comment:
@brycemcd I've updated the rake task so that it selects the last user and creates a new heart_rate_data entry for the current day